region/crateria/central.json

```diff
@@ -406,7 +411,8 @@
                   "canDestroyBombWalls",
                   {"canComeInCharged": {
                     "fromNode": 1,
-                    "framesRemaining": 0
+                    "framesRemaining": 0,
+                    "shinesparkFrames": 75
```

Comment:
If you can come in with 180 frames, you can also get through without sparking

Reply:
Ah yes, I somehow thought that was already added.